### ISSUE TITLE
⛔ 🚀 Alt text is available for the Basic Gallery widget, but it is absent from the Pro Gallery counterpart.

### ISSUE THREAD

**Issue by losvincent** [elementor/elementor#34102]
### Description

Images in the Basic Gallery widget are rendered as standard <img> tags with alt text attributes included. In contrast, but when update to elementor pro version,and insert Pro Gallery widget,it outputs images as background images instead, which results in the absence of SEO-friendly alt text.

![Image](https://github.com/user-attachments/assets/9956116c-5d9c-4212-8497-888b41e8ea57)

![Image](https://github.com/user-attachments/assets/af2158a9-6d06-4e85-a95d-37d0dec6b124)

### Steps to reproduce

1.add basic gallery and insert images with alt set in media library
2.add pro gallery and insert images with alt set too
3.inspect in browser front code

### Expected behavior

Set the same seo friendly alt tags to pro gallery widget

### Elementor System Info

<details>
<summary>Click to reveal</summary>

```txt
Server Environment		
Operating System:	Linux	
Software:	nginx/1.28.0	
MySQL version:	Source distribution v8.4.0	
PHP Version:	8.4.14	
PHP Memory Limit:	568M	
PHP Max Input Vars:	1000	
PHP Max Post Size:	50M	
GD Installed:	Yes	
ZIP Installed:	Yes	
Write Permissions:	All right	
Elementor Library:	Connected	
WordPress Environment		
Version:	6.9	
WP Multisite:	No	
Max Upload Size:	50 MB	
Memory limit:	40M	
Max Memory limit:	568M	
Permalink Structure:	/%postname%/	
Language:	en_US	
Timezone:	0		
Theme		
Name:	Astra Child	
Version:	1.0.0	
Author:	Brainstorm Force	
Child Theme:	Yes	
Parent Theme Name:	Astra	
Parent Theme Version:	4.11.18	
Parent Theme Author:	Brainstorm Force	
User		
Role:	administrator	
WP Profile lang:	en-US	
User Agent:	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/143.0.0.0 Safari/537.36 Edg/143.0.0.0	
Active Plugins		
Advanced Custom Fields - 6.7.0	By WP Engine	
Chaty - 3.5.1	By Premio	
Elementor - 3.34.0	By Elementor.com	
Elementor Pro - 3.34.0	By Elementor.com	
FileBird Lite - 6.5.2	By Ninja Team	
Rank Math SEO - 1.0.260	By Rank Math SEO	
Redis Object Cache - 2.7.0	By Till Krüss	
WP Armour - Honeypot Anti Spam - 2.3.04	By Dnesscarkey	
WPB Accordion Menu or Category - 1.8.3	By WPBean

```
</details>

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by nicholaszein** [comment]
Hello @losvincent.

Thank you for reaching out!

This is more of a feature request, not a bug.

The widget in question was never intended to include images with alt tags when it was initially conceived, even though we do understand now, years later, the importance of accessibility. It was built to ensure specific layouts, and at the time the only efficient and reliable way of achieving those layouts was to use background images instead.

We are aware this widget would require some accessibility improvements, however, due to backward compatibility, we cannot make such drastic changes without really having the community accepting the breaking changes. This means we will only make these changes if there is enough demand from the community.

Otherwise we still, despite understanding the importance of this functionality, cannot treat it as a bug. There is a difference between intended and expected behavior.

Please, if you think this should be implemented, [search for existing feature requests](https://elemn.to/gh-feature-requests) on the same subject, and vote for them.

Please, help keep this space organized. Only post valid issues, and use the appropriate channels for requesting features.

Thank you for understanding.

Kind regards
